bin/inx-chronicle/src/launcher.rs

```diff
@@ -0,0 +1,243 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! Contains the `Launcher` actor.
```

Comment:
While we're at it, maybe we can write a sentence or two about the purpose of this actor.